src/transforms/ember-getWithDefault-to-lodash-get/tests/transform.test.ts

```diff
@@ -0,0 +1,37 @@
+import runTestCases from '../../../utils/testHelper'
```

Comment:
Can use the `#` subpath import instead to avoid relative import paths
```suggestion
import runTestCases from '#utils/testHelper.js'
```

Reply:
Done